### PR TITLE
Hashmap repeated word

### PR DESCRIPTION
**Pull request by eden-brekke**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
 - [x] Summary, Description, Approach & Efficiency, Solution
 - [x] Picture of whiteboard
 - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
 - [x] “Happy Path” - Expected outcome
 - [x] Expected failure
 - [x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: python/code_challenges/hashtable_repeated_word.py [hunk 1]
```diff
@@ -1,5 +1,13 @@
-from data_structures.hashtable import Hashtable
+import re
 
+def first_repeated_word(word):
```
Comment (chloenott) [inline python/code_challenges/hashtable_repeated_word.py:3]: There might be an opportunity to choose a clearer input name, like `text`, `sentence`, `phrase`, or `words`. Though `words` is reused below, so I'd be a little iffy on using the same variable name when the datatype changes from string to list; though maybe `words_string` and `words_list` is a compromise.

File: python/code_challenges/hashtable_repeated_word.py [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+    regex = re.compile('[^a-zA-Z ]')
+    stripped_words = regex.sub('', word)
+    words = stripped_words.lower().split()
 
-def first_repeated_word():
-    pass
+    dictionary = set()
```
Comment (chloenott) [inline python/code_challenges/hashtable_repeated_word.py:8]: Similar idea to above choosing variable names that maximize readability for `dictionary`. Maybe `words_checked`?

**Review by chloenott — COMMENTED** [review]
Nice, clean code! The tests are really thorough too.
